zenoh/session.py

```diff
@@ -29,9 +29,10 @@ class Publisher:
     def __init__(self, p: _Publisher):
         self._inner_ = p
 
-    def put(self, value: IntoValue, encoding: Encoding = None):
-        "An optimised version of ``session.put(self.key_expr, value, encoding=encoding)``"
-        self._inner_.put(Value(value, encoding))
+    def put(self, value: IntoValue, encoding: Encoding = None, attachment: IntoAttachment = None):
+        "An optimised version of ``session.put(self.key_expr, value, encoding=encoding, attachment=attachment)``"
```

Comment:
Although is not originated from this PR, I find the comment a bit confusing. What does `an optimised version of` mean here?

Reply:
I don't know 🤷‍♂️